pyproject.toml

```diff
@@ -21,6 +21,7 @@ mkl = "2021.4.0"
 geopandas = "^0.14.2"
 matplotlib = "^3.6.3"
 shapely = "^2.0.2"
+pydantic = "^1.9.0"
```

Comment:
Why not use the latest pydantic? The latest version is already 2.6 :)

Reply:
will update